code/modules/newscaster/obj/newspaper.dm

```diff
@@ -114,7 +114,8 @@
 					dat += "<br><i>There is a small scribble near the end of this page... It reads: \"[scribble]\"</i>"
 				dat+= "<hr><div style='float:left;'><a href='?src=[UID()];prev_page=1'>Previous Page</a></div>"
 			else
-				dat += "i'm sorry to break your immersion. This shit's bugged. Report this bug to Agouri, polyxenitopalidou@gmail.com"
+				// No trailing punctuation so that it's easy to copy and paste the address
+				dat += "We're sorry to break your immersion, but there has been an error with the newscaster. Please report this error, along with any more information you have, to https://github.com/ParadiseSS13/Paradise/issues/new?assignees=&labels=&template=bug_report.md&title="
```

Comment:
`config.githuburl` can be used here instead.

Reply:
My only concern there is that we could wind up sending non-programmers to a big scary Github page if we do that.
If there's a consensus, I could create `config.githuperrorreporturl` instead. (Although I'd prefer `config.githubErrorReportUrl`.)

Reply:
That would be good. Could also be used by admins/mentors to forward people to make an issue etc.
I'd recommend using snake_case for the naming btw

Reply:
There is a config freeze due to the ongoing conversion to TOML. I'm going with AffectedArc's solution instead